pallets/associated-accounts/src/lib.rs

```diff
@@ -75,16 +75,16 @@ pub mod pallet {
     #[pallet::event]
     #[pallet::generate_deposit(pub(super) fn deposit_event)]
     pub enum Event<T: Config> {
-        /// Altvr username updated  \[owner, username\]
-        AltVRUsernameChanged(T::AccountId, ternoa::String),
+        /// Altvr username created  \[owner, username\]
+        AltVRUsernameCreated(T::AccountId, ternoa::String),
```

Comment:
You can create something  only once but you can change a name multiple times. Since you can always update your username I think that `Changed` would be a better fit than `Created`. What are your thoughts?

Reply:
@markopoloparadox  The why of the suffix created is that we are actually creating it and make sense to let Created.
If the plan is just set a username the it makes sense to let UsernameCreated once the user Update that username we can then use AltVRUsernameChanged.

Reply:
Having two events for one extrinsic would be too much. We can stick with the `AltVRUsernameCreated` if you want.